MatrixSDK/MXSession.m

```diff
@@ -1576,7 +1580,9 @@ - (void)setAntivirusServerURL:(NSString *)antivirusServerURL
     // Update the media manager
     [mediaManager setAntivirusServerURL:antivirusServerURL];
     
-    // TODO: configure here a scan manager.
+    // Configure scan manager
```

Comment:
You may mention here that the scan manager will be nil if the provided url is nil